Aztec/Classes/Libxml2/DOM/Data/TextNode.swift

```diff
@@ -216,6 +216,10 @@ extension Libxml2 {
         // MARK: - EditableNode
 
         func append(_ string: String) {
+            if !shouldSanitizeText() {
```

Comment:
No biggie but this could look nicer with a `guard`.

```swift
guard shouldSanitizeText() else {
    append(sanitizedString: string)
    return
}
```